presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/GeoFunctions.java

```diff
@@ -575,14 +464,35 @@ public static Double lineLocatePoint(@SqlType(GEOMETRY_TYPE_NAME) Slice lineSlic
         return new LengthIndexedLine(line).indexOf(point.getCoordinate()) / line.getLength();
     }
 
+    @Description("Returns the point in the line at the fractional length.")
+    @ScalarFunction("line_interpolate_point")
+    @SqlType(GEOMETRY_TYPE_NAME)
+    public static Slice lineInterpolatePoint(@SqlType(GEOMETRY_TYPE_NAME) Slice lineSlice, @SqlType(DOUBLE) double fraction)
+    {
+        if (!(0.0 <= fraction && fraction <= 1.0)) {
+            throw new PrestoException(INVALID_FUNCTION_ARGUMENT, format("line_interpolate_point: Fraction must be between 0 and 1, but is %s", fraction));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code uses &#39;%s&#39; to format double: fraction (declared at line 470). This is a potential locale-sensitive handling issue. It might cause errors in the handling and processing of the statement at line: 473. Consider formatting this data with &#39;%f&#39; instead.